docs/sources/flow/monitoring/agent-resource-usage.md

```diff
@@ -0,0 +1,72 @@
+---
+aliases:
+- /docs/agent/flow/monitoring/resource-usage/
+- /docs/grafana-cloud/agent/flow/monitoring/resource-usage/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/monitoring/resource-usage/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/monitoring/resource-usage/
+- /docs/grafana-cloud/send-data/agent/flow/monitoring/resource-usage/
+canonical: https://grafana.com/docs/agent/latest/flow/monitoring/resource-usage/
+description: Guidance for expected Agent resource usage
+headless: true
+title: Resource usage
+---
+
+# {{% param "PRODUCT_NAME" %}} resource usage
+This page provides guidance for expected resource usage of {{% param "PRODUCT_NAME" %}}
+for each telemetry type, based on operational
+experience of some of the {{% param "PRODUCT_NAME" %}} maintainers.
+
+{{% admonition type="note" %}}
+
+The resource usage depends on the workload, hardware and the configuration used.
+The information on this page is a good starting point for 
+most users, but your actual usage may be different.
+
+{{% /admonition %}}
+
+## Prometheus metrics
+
+The Prometheus metrics resource usage depends mainly on the number of active series
+that need to be scraped.
+
+As a rule of thumb, **per each 1 million active series** with 15s scrape interval,
```

Comment:
We should use 60s here since that is what we recommend everywhere.

Reply:
Hmm, in that case we don't have a data from production... unless we change our prod to match what we recommend. Any idea why do we not use the recommended interval?

Reply:
AFAIK 60s is optimized for Grafana Cloud and is documented here https://grafana.com/docs/grafana-cloud/cost-management-and-billing/understand-your-invoice/metrics-invoice/#data-points-per-minute-dpm. It is possible to tweak it via scrape config but it has a direct impact on cost

Reply:
I guess the best way forward is for me to set up an experiment where we do a 60s scrape interval for some time to collect data?

Reply:
We can try and see what we get, I assume most OSS users will use the 15s interval and Cloud users would prefer the 60s interval. My guess is that memory usage should be similar, but CPU and memory will change (more data points to filter and send)